packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -480,6 +592,32 @@ describe('connectInfiniteHits', () => {
       expect(res2.hasMore).toBe(true);
     });
 
+    it('prepend hits internally', () => {
+      const context = createMultiIndexContext();
+      const getProvidedProps = connect.getProvidedProps.bind(context);
+
+      const hits = [{}, {}];
+      const hits0 = [{}, {}];
+
+      const res1 = getProvidedProps(null, null, {
```

Comment:
We could re-use the same name used for the single index, it's easier to understand.